apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -133,6 +133,22 @@ const audioStyle = {
   }
 };
 
+const audioStyleRTL = {
+  wrapper: {
+    float: 'left'
+  },
+  button: {
+    height: 24,
+    marginTop: '3px',
+    marginBottom: '3px'
+  },
+  buttonImg: {
+    lineHeight: '24px',
+    fontSize: 15,
+    paddingLeft: 12
+  }
+};
+
```

Comment:
yeah! Thanks for including these styles. RTL doesn't get enough love on our site